packages/turf-point-grid/index.js

```diff
@@ -1,18 +1,17 @@
 import distance from '@turf/distance';
-import turfBBox from '@turf/bbox';
-import { point, featureCollection, isObject } from '@turf/helpers';
+import {point, featureCollection, isObject, isNumber} from '@turf/helpers';
 import inside from '@turf/inside';
-import { getType } from '@turf/invariant';
+import {getType} from '@turf/invariant';
 
 /**
  * Creates a {@link Point} grid from a bounding box, {@link FeatureCollection} or {@link Feature}.
  *
  * @name pointGrid
- * @param {BBox|GeoJSON} bbox extent in [minX, minY, maxX, maxY] order
- * @param {number} cellSide the distance between points
+ * @param {Array<number>} bbox extent in [minX, minY, maxX, maxY] order
+ * @param {number} cellSide the distance between points, in units
  * @param {Object} [options={}] Optional parameters
  * @param {string} [options.units="kilometers"] used in calculating cellSide, can be degrees, radians, miles, or kilometers
- * @param {number} [options.bboxIsMask=false] if true, and bbox is a Polygon or MultiPolygon, the grid Point will be created
+ * @param {Feature<Polygon|MultiPolygon>} [options.mask=undefined] if passed a Polygon or MultiPolygon, the grid Points will be created only inside it
```

Comment:
No need to define the option as `undefined`, `[options.mask]` is perfectly fine.